docs/standard/modern-web-apps-azure-architecture/architectural-principles.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Architectural principles | Microsoft Docs 
+description: Architect Modern Web Applications with ASP.NET Core and Azure | Architectural principles
+keywords: Docker, Microservices, ASP.NET, Container
+author: ardalis
+ms.author: wiwagn
+ms.date: 10/06/2017
```

Comment:
ms.product and ms.technology: do you need those?

Reply:
updated